stable/efs-provisioner/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: efs-provisioner
 description: A Helm chart for the AWS EFS external storage provisioner
-version: 0.1.1
+version: 0.1.2
 appVersion: v0.1.2
```

Comment:
```suggestion
appVersion: v2.1.0-k8s1.11
```

Reply:
Hi @kppullin, I suggest you update the `appVersion` to the latest version you have tested with.
https://quay.io/repository/external_storage/efs-provisioner